package.json

```diff
@@ -47,20 +51,22 @@
     "react-dom": "^17.0.0",
     "react-redux": "^7.1.0",
     "react-test-renderer": "^17.0.0",
-    "redux": "^3.7.2 || ^4.0.3"
+    "redux": "^3.7.2 || ^4.0.3",
+    "typescript": "^4.5.4"
   },
   "peerDependencies": {
     "react": "^16.9.0 || ^17.0.0",
     "react-dom": "^16.9.0 || ^17.0.0"
   },
   "scripts": {
-    "build": "babel src -d lib --ignore '*.test.js'",
+    "build": "babel src -d lib --extensions .ts,.tsx --ignore '*.test.js'",
     "eslint-check": "eslint-config-prettier .eslintrc.js",
-    "fix": "eslint --ignore-pattern 'lib/*' --ext .js,.md --fix .",
-    "lint": "eslint --ignore-pattern 'lib/*' --ext .js,.md .",
+    "fix": "eslint --ignore-pattern 'lib/*' --ext .ts,.tsx,.js,.md --fix .",
+    "lint": "eslint --ignore-pattern 'lib/*' --ext .ts,.tsx,.js,.md .",
     "test": "jest src",
+    "test:types": "tsc",
```

Comment:
This is not really a command in the `test` namespace IMO.
Maybe a `compile` command instead?

Reply:
I was inspired by react-query for that name

Hmm it's not really compiling, more like a check. Maybe something like check:ts or check:types ?

Naming is hard!

Reply:
It seems it is the choice of Next.js too: https://github.com/vercel/next.js/blob/34dcede6902fc1bc100a3a9d8874851c4bcd9977/package.json#L14.
OK then!